docs/api/menu-item.md

```diff
@@ -12,7 +12,7 @@ See [`Menu`](menu.md) for examples.
   * `click` Function (optional) - Will be called with
     `click(menuItem, browserWindow, event)` when the menu item is clicked.
     * `menuItem` MenuItem
-    * `browserWindow` [BrowserWindow](browser-window.md)
+    * `browserWindow` [BrowserWindow](browser-window.md) | undefined - This will not be defined if no window is open.
```

Comment:
Should use `(optional)` to define an optional type

Reply:
@MarshallOfSound docs-parser currently precludes using `optional` for a variable followed by a non-optional variable, and switching them would be a breaking change which is why i did it this way

Reply:
TS might preclude it but our docs parser doesn't and is smart enough to generate this correctly if you use (optional)

Reply:
resolved - this is not in fact possible 😛